projects/ccf-rui/src/app/modules/registration-modal/registration-content/registration-content.component.ts

```diff
@@ -54,7 +55,15 @@ export class RegistrationContentComponent {
     readonly model: ModelState,
     public dialogRef: MatDialogRef<RegistrationContentComponent>
   ) {
+    page.user$.subscribe(user => this.checkNameValid(user));
```

Comment:
You should call `cdr.markForCheck()` in these subscribe calls to be on the safe side